README.md

```diff
@@ -524,16 +530,27 @@ using (var tracerFactory = TracerFactory.Create(builder => builder
 #### Traces
 
 ```csharp
-using (var tracerFactory = TracerFactory.Create(builder => builder
-    .AddProcessorPipeline(c => c.SetExporter(new StackdriverTraceExporter("YOUR-GOOGLE-PROJECT-ID")))))
-{
-    var tracer = tracerFactory.GetTracer("stackdriver-test");
-    var span = tracer
-        .SpanBuilder("incoming request")
-        .StartSpan();
+var spanExporter = new StackdriverTraceExporter(projectId);
+
+using var tracerFactory = TracerFactory.Create(builder => builder.AddProcessorPipeline(c => c.SetExporter(spanExporter)));
+var tracer = tracerFactory.GetTracer("stackdriver-test");
+
+DistributedContext.Carrier = AsyncLocalDistributedContextCarrier.Instance; // Enable asynclocal carrier for the context

```

Comment:
I'm not sure we should be configuring a custom DistributedContext in these examples, this is a separate to the example which is trying to demonstrate the StackDriverExporter.

Reply:
should i remove?

Reply:
Did you add it for a particular reason? If not, I think we can remove to simplify the examples.

Reply:
I see. What do you think about this:
```csharp
var spanExporter = new StackdriverTraceExporter(projectId);

using var tracerFactory = TracerFactory.Create(builder => builder.AddProcessorPipeline(c => c.SetExporter(spanExporter)));
var tracer = tracerFactory.GetTracer("stackdriver-test");

using (tracer.StartActiveSpan("/getuser", out TelemetrySpan span))
{
    span.AddEvent("Processing video.");
    span.PutHttpMethodAttribute("GET");
    span.PutHttpHostAttribute("localhost", 8080);
    span.PutHttpPathAttribute("/resource");
    span.PutHttpStatusCodeAttribute(200);
    span.PutHttpUserAgentAttribute("Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:47.0) Gecko/20100101 Firefox/47.0");

    Thread.Sleep(TimeSpan.FromMilliseconds(10));
}
```